### PR TITLE
✨Source Retently: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #28118: Retently

### PR DESCRIPTION
**Pull request by avirajsingh7**
## What
Migrating Source retently to Low-Code CDK
Closes https://github.com/airbytehq/airbyte/issues/28118

## How
Developed using (Configuration Based Source) low-code CDK

## Recommended reading order

1. manifest.yaml
2. schemas/*

 ##  Test
<details>
<summary>Acceptance Test</summary>
Screenshot 2023-08-03 at 6 52 13 PM
</details>

# 🚨 User Impact 🚨
**No breaking changes**, migration to low-code
<details>
<summary>Migrate Connector : Source retently</summary>

- [x] **Community Member? Grant Edit Access to Maintainers [instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)**
- [x] **Secrets in the Connector's Spec**
- [x] **Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run ./gradlew :airbyte-integrations:connectors:<name>:integrationTest.**
- [x] **Code Reviews Completed**
  - [x] **Documentation Updated**
  - [x] **Connector's README.md**
  - [x] **Connector's bootstrap.md**
  - [x] **docs/integrations/<source or destination>/<name>.md (including changelog. See changelog example)**
  - [x] **docs/integrations/README.md**
  - [x] **airbyte-integrations/builds.md**
- [x] **PR Name follows PR Naming Conventions**
</details>

### CODE REVIEW

**Comment by sajarin** [comment]
Thanks for the contribution! Can you resolve the merge conflict @avirajsingh7?

**Comment by avirajsingh7** [comment]
@sajarin sorry of that, resolved now

File: airbyte-integrations/connectors/source-retently/acceptance-test-config.yml [hunk 1]
```diff
@@ -4,28 +4,23 @@ connector_image: airbyte/source-retently:dev
...
-        status: "succeed"
-      - config_path: "secrets/config_oauth.json"
-        status: "succeed"
       - config_path: "integration_tests/invalid_config.json"
         status: "failed"
   discovery:
     tests:
       - config_path: "secrets/config.json"
-        backward_compatibility_tests_config:
-          disable_for_version: "0.1.5"
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-retently/acceptance-test-config.yml:22]: Some of the changes you've made to some of the schema files is resulting in backwards incompatibility. Can you disable these tests for the prior version?

File: airbyte-integrations/connectors/source-retently/acceptance-test-config.yml [hunk 2]
```diff
@@ -4,28 +4,23 @@ connector_image: airbyte/source-retently:dev
...
   basic_read:
     tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
-        empty_streams:
-          - name: reports
+        empty_streams: []
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-retently/acceptance-test-config.yml:21]: templates is an empty stream in our sandbox account

**Review by sajarin — CHANGES_REQUESTED** [review]
looks good, some minor comments after running tests locally

**Comment by avirajsingh7** [comment]
@sajarin can you check now?

File: airbyte-integrations/connectors/source-retently/icon.svg
Comment (sajarin) [inline airbyte-integrations/connectors/source-retently/icon.svg:1]: Why was this deleted?

Comment (avirajsingh7) [reply airbyte-integrations/connectors/source-retently/icon.svg:1]: my bad, added now

**Comment by sajarin** [comment]
Thanks for the PR @avirajsingh7! This has been merged and tested in #29164. The reason why I wasn't able to merge this PR was because I needed to push some changes and run them against our CI but couldn't push to your branch since I didn't have branch permissions. Nevertheless, I forked your changes and tested in the PR aforementioned.

I'm closing this PR but this should still count for the hackathon of course. fyi @RealChrisSean

**Comment by avirajsingh7** [comment]
thanks @sajarin

**Pull request closed without merging**